de.peeeq.wurstscript/src/main/java/de/peeeq/wurstio/languageserver/requests/RunTests.java

```diff
@@ -366,7 +369,7 @@ public void showInfoMessage(String message) {
     }
 
     private class TestTimeOutException extends Throwable {
-
+        private static final long serialVersionUID = -7080085729129882874L;
```

Comment:
?!

Reply:
It's a generated value for a serialisable class. IDE flagged it, I'm not sure if it's actually needed or not

Reply:
> It's a generated value for a serialisable class

Is this class serialized?

Reply:
I don't know sorry, but my language server flagged it, so maybe yes?